backend/src/bundles/auth/auth.controller.ts

```diff
@@ -32,6 +34,11 @@ import { type ILogger } from '~/common/logger/logger.js';
 
 import { type AuthService } from './auth.service.js';
 
+interface JwtPayload {
```

Comment:
Consider changing to a type

![image](https://github.com/BinaryStudioAcademy/bsa-winter-2023-2024-resumegemm/assets/92567020/01d25ca6-7bfc-4ce8-aa64-53e92a24ba5e)